packages/react/src/components/accordion/accordion-container.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, ReactElement, useMemo, createRef, useRef } from 'react';
+import { AccordionProps, AccordionContainerProps } from './accordion-types';
+import { Accordion } from './accordion';
+import { StyledAccordionGroup } from './accordion-styles';
+
+export const AccordionContainer: React.FC<AccordionContainerProps> = ({
+    mode = 'single', children, defaultExpandedItemIds, disabledItemIds = [],
+}) => {
+    const [expandedItemIds, setExpandedItemIds] = useState<string[]>(defaultExpandedItemIds || []);
+    const focusedAccordionRef = useRef<number | null>(null);
+
+    const handleToggle = (itemId: string): void => {
+        if (mode.toLowerCase() === 'single') {
```

Comment:
At the development time it's right (TS won't allow and will show the error), but at the implementation time if someone type Multi or Single the accordion will be displayed correctly but the functionality of multi open wont work.
That's why I added it.
Even knowing this do you think I should remove it?

Reply:
If their project has TS, they would have the type check as well since our NPM includes the typing as well. 

But let suppose they don't have TS, in that case, if they see in the docs that this attribute accept the values "single" or "multi", they should not write "SINGLE" and we have no reason to accept it.  You can see the same behavior with MUI: https://codesandbox.io/s/shy-currying-9tfwcg?file=/Demo.tsx

Also all our other components are case sensitive as well, what make this one more special?

Reply:
In this scenario, my experience in accessibility, coupled with my own understanding of dyslexia, has made me aware that even a small typo can lead to significant issues. Considering this, I've taken steps to ensure the end user's experience with this product remains uncomplicated.

For instance, if someone accidentally types "mode="Multi"" instead of the correct "mode="multi"," I've implemented a solution that automatically corrects this typo error to enhance usability. It's important to note that this feature is designed for the end user of the product, rather than the developer working on the design system.

Please let me know if I should remove this.

Reply:
I'm with Savut on this too. There would be no end to what we can do to fix the developer's mistake. I see the props on our components as a contract to the DS API. If the contract isn't respected, then I would expect the component to "fail".

Besides, all/most of the new projects should be in Typescript, so the compiler should throw an error when the prop doesn't match any acceptable value.